app/services/irs_attempts_api/attempt_event.rb

```diff
@@ -26,6 +26,7 @@ def to_jwe
         zip: 'DEF',
         alg: 'RSA-OAEP',
         enc: 'A256GCM',
+        kid: JWT::JWK.new(event_data_encryption_key).kid,
```

Comment:
This will probably allow us to remove the `irs_attempt_api_public_key_id` parameter later: https://github.com/18F/identity-idp/blob/main/config/application.yml.default#L347

Though that's a breaking schema change so we should hold off.